ocs_ci/utility/retry.py

```diff
@@ -64,6 +64,10 @@ def deco_retry(f):
         @wraps(f)
         def f_retry(*args, **kwargs):
             mtries, mdelay = tries, delay
+            logger.debug(
+                f"Executing {f.__name__}. Tries: {mtries}. Delay: {mdelay}. Backoff: {backoff}"
```

Comment:
we can probably print this one only on first retry as there can be case where we will not re-try at all